fastly.services.yml

```diff
@@ -16,7 +16,7 @@ services:
     tags:
       - { name: event_subscriber }
   fastly.cache_tags.add_soft_purge_headers:
```

Comment:
Would you want to rename the service to ` fastly.cache_tags.add_stale_headers` and update anywhere that called the old service name, just for consistency?